pom.xml

```diff
@@ -376,7 +375,20 @@
         <os.bitness>64</os.bitness>
         <!-- skip test on cross-compile -->
         <maven.test.skip>true</maven.test.skip>
-        <cmake.generate.args>-DCMAKE_TOOLCHAIN_FILE=toolchain/Aarch64.cmake -DNATIVE_LIB_DIR=aarch64</cmake.generate.args>
+        <cmake.generate.args>-DCMAKE_TOOLCHAIN_FILE=toolchain/Aarch64.cmake -DNATIVE_LIB_DIR=${os.arch}_${os.bitness}</cmake.generate.args>
+      </properties>
+    </profile>
+
+    <!-- powerpc profile -->
+    <profile>
+      <id>ppcle</id>
```

Comment:
Same here, other OSes might as well be low endian on PPC. Also, it should be ppc64le to comply with the other tools we use.